templates/layout.html

```diff
@@ -8,6 +8,7 @@
 <link rel="stylesheet" type="text/css" href="vendor/icomoon/style.css" />
 <link rel="stylesheet" type="text/css" href="vendor/bootstrap-datepicker-1.4.0/css/bootstrap-datepicker3.min.css" />
 <script type="text/javascript" src="vendor/jquery-2.1.1.min.js"></script>
+<script src='https://www.google.com/recaptcha/api.js'></script>
```

Comment:
Include external libs only if recaptcha is enabled and only on pages where it is used. #142